src/mito2/src/memtable/time_series.rs

```diff
@@ -237,39 +253,44 @@ impl Series {
         self.freeze(region_metadata);
 
         let mut frozen = self.frozen.clone();
-        let values = if frozen.len() == 1 {
+        let values = if frozen.is_empty() {
+            Values::new_empty(region_metadata)
```

Comment:
Is it possible that frozen is empty? A series in the series set should at least contain one point. I think we could also add an assertion.

Reply:
No, given that we cannot delete rows from memtable, each series must contain at least one row and hence one frozen value.